.github/workflows/build.yml

```diff
@@ -106,7 +106,7 @@ jobs:
 
       - name: Set latest tag
         id: setlatest
-        if: ${{ (github.event_name == 'release' && github.event.release.target_commitish == 'master') || (github.event_name == 'workflow_dispatch' && github.event.inputs.push_image == 'true' && github.event.inputs.use_latest_tag == 'true') }}
+        if: ${{ (github.event_name == 'release' && github.event.release.target_commitish == 'master') || (github.event_name == 'workflow_dispatch' && github.event.inputs.use_latest_tag == 'true') }}
```

Comment:
Is there any practical difference?

Reply:
Yes, it allows to build image with `latest` tag without pushing it to the registry.

Reply:
So if I run this workflow manually, I can check the building of the `latest` image.